platform/android/java/build.gradle

```diff
@@ -206,79 +107,95 @@ def generateBuildTasks(String flavor = "template") {
     String libsDir = isTemplate ? "lib/libs/" : "lib/libs/tools/"
     for (String target : supportedFlavorsBuildTypes[flavor]) {
         File targetLibs = new File(libsDir + target)
+
         if (!excludeSconsBuildTasks || (targetLibs != null
             && targetLibs.isDirectory()
             && targetLibs.listFiles() != null
             && targetLibs.listFiles().length > 0)) {
+
             String capitalizedTarget = target.capitalize()
             if (isTemplate) {
-                // Copy the generated aar library files to the build directory.
-                tasks += "copy${capitalizedTarget}AARToAppModule"
-                // Copy the generated aar library files to the bin directory.
-                tasks += "copy${capitalizedTarget}AARToBin"
-                // Copy the prebuilt binary templates to the bin directory.
-                tasks += "copy${capitalizedTarget}BinaryToBin"
+                // Copy the Godot android library archive file into the app module libs directory.
+                // Depends on the library build task to ensure the AAR file is generated prior to copying.
+                String copyAARTaskName = "copy${capitalizedTarget}AARToAppModule"
+                if (tasks.findByName(copyAARTaskName) != null) {
+                    buildTasks += tasks.getByName(copyAARTaskName)
+                } else {
+                    buildTasks += tasks.create(name: copyAARTaskName, type: Copy) {
+                        dependsOn ":lib:assembleTemplate${capitalizedTarget}"
+                        from('lib/build/outputs/aar')
+                        include("godot-lib.template_${target}.aar")
```

Comment:
This won't copy the `dev` builds that have a name like: `godot-lib.template_debug.dev.aar`.

Could do something like:

```gradle
String targetSuffix = target
if (target == "dev") {
    targetSuffix = "debug.dev"
}
```